packages/components/organisms/f-checkout/src/components/_tests/Checkout.test.js

```diff
@@ -257,6 +259,36 @@ describe('Checkout', () => {
                 expect(spinner.exists()).toBe(true);
             });
         });
+
+        describe('nonFulfillableError', () => {
+            describe('when `nonFulfillableError.openInDialog` is `true`', () => {
+                it('should show a mega modal displaying the error title and description', () => {
+                    // Arrange & Act
+                    const fulfilmentTimeIssue = CheckoutIssues[ERROR_CODE_FULFILMENT_TIME_UNAVAILABLE];
+
+                    const wrapper = mount(VueCheckout, {
+                        i18n,
+                        store: createStore(),
+                        localVue,
+                        propsData,
+                        data () {
+                            return {
+                                nonFulfillableError: fulfilmentTimeIssue
+                            };
+                        }
+                    });
+
+                    const errorModal = wrapper.find('[data-test-id="checkout-issue-modal"]');
+                    const errorTitle = wrapper.find('[data-test-id="checkout-issue-modal-title"]');
+                    const errorMessage = wrapper.find('[data-test-id="checkout-issue-modal-message"]');
+
+                    // Assert
+                    expect(errorModal.exists()).toBe(true);
+                    expect(errorTitle.exists()).toBe(true);
+                    expect(errorMessage.exists()).toBe(true);
```

Comment:
Should we also test the content and not just that exists?

Reply:
This was a question I wanted to ask you. All tests I've seen which test the value of text are based on the text being a prop value. How do I test text from translations?